clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx

```diff
@@ -30,29 +33,48 @@ const ALL_COLUMNS: ColumnMetadata[] = [
   { name: "Resource ID", attribute: "fidesctl_meta.resource_id" },
 ];
 
-const ScanResultsForm = () => {
+interface Props {
+  manualSystemSetupChosen: boolean;
+}
+
+const ScanResultsForm = ({ manualSystemSetupChosen }: Props) => {
   const systems = useAppSelector(selectSystemsForReview);
   const dispatch = useAppDispatch();
+  const router = useRouter();
   const {
     isOpen: isWarningOpen,
     onOpen: onWarningOpen,
     onClose: onWarningClose,
   } = useDisclosure();
+  const [createSystem] = useCreateSystemMutation();
   const [selectedSystems, setSelectedSystems] = useState<System[]>(systems);
+  const features = useFeatures();
   const [selectedColumns, setSelectedColumns] =
     useState<ColumnMetadata[]>(ALL_COLUMNS);
 
+  const createSystems = async () => {
+    await selectedSystems.forEach((system: System) => createSystem(system));
+  };
+
   const confirmRegisterSelectedSystems = () => {
-    dispatch(chooseSystemsForReview(selectedSystems.map((s) => s.fides_key)));
-    dispatch(changeStep());
+    if (manualSystemSetupChosen) {
+      dispatch(chooseSystemsForReview(selectedSystems.map((s) => s.fides_key)));
+      return dispatch(changeStep());
+    }
```

Comment:
why do we care about if `manulSystemSetupChosen`? doesn't the manual step skip the scan results step (since there are no scan results?)